bson/objectid.py

```diff
@@ -67,16 +40,23 @@ def _raise_invalid_id(oid):
         " or a 24-character hex string" % oid)
 
 
+def _random_bytes():
+    """Get the 5-byte random field of an ObjectId."""
+    return struct.pack(">Q", SystemRandom().randint(0, 0xFFFFFFFFFF))[3:]
```

Comment:
Nice! Didn't know about `getrandbits`. Is there a reason you'd prefer using this over what I have currently?

Reply:
Nah, doesn't matter.

Reply:
I asked because I'd like to use `getrandbits` for initializing the counter as well if there was a rationale for it... Let's leave it as-is for now.

Reply:
My rationale was that  `getrandbits(40)` is easier to read than `randint(0, 0xFFFFFFFFFF)`.

Reply:
But far less easy to read than `sys.urandom(5)`, directly returning five bytes with no need to encode, which internally invokes SystemRandom.  (Until seeing this, I was actually unaware of the existence of that class!)

Reply:
Thanks for the tip @amcgregor. It is a bit silly that we're using struct.pack and SystemRandom here instead of just `sys.urandom(5)`...

Would you like to open a PR for the change?

Reply:
> Would you like to open a PR for the change?

Alas, due to historical reasons (ref: SERVER-25717), I would prefer not to.

Reply:
I fixed this in https://github.com/mongodb/mongo-python-driver/commit/87793a9058108c18bbe2aa0112cde9d98768c52d. Thanks again!